### PR TITLE
View all upcoming appointments

### LINKED ISSUE #24: As a user, I can view all upcoming appointments displayed in a concise and accessible format.

### PR DESCRIPTION
**Pull request by xuelinglow**
Screenshot 2024-03-18 013334

Currently looks like this, let me know if you guys have any suggestions regarding the ui design! 
Also tried to made it sort by date and time

### CODE REVIEW

**Review by ThatGirlJam — APPROVED** [review]
LGTM 👍

File: src/test/java/seedu/address/logic/commands/AddAppCommandTest.java [hunk 1]
```diff
@@ -160,7 +160,7 @@ public boolean hasPersonWithNric(Nric nric) {
 
         @Override
         public Person getPersonWithNric(Nric nric) {
-            throw new AssertionError("This method should not be called.");
+            return null;
         }
```
Comment (limrik) [inline src/test/java/seedu/address/logic/commands/AddAppCommandTest.java:164]: Is this method duplicated? Seems to also be over-riden below in lines 271-279.

**Review by limrik — APPROVED** [review]

File: src/test/java/seedu/address/logic/commands/AddAppCommandTest.java (hunk 1 above)
Comment (ThatGirlJam) [reply src/test/java/seedu/address/logic/commands/AddAppCommandTest.java:164]: I think there's 2 versions of the model stub, but not too sure what theyre used for either

Comment (xuelinglow) [reply src/test/java/seedu/address/logic/commands/AddAppCommandTest.java:164]: Oh yeah i changed this when i was doing the getPersonWithNric in AppCommand, will change it back, thanks for the review!

**Pull request merged**